index.html

```diff
@@ -1705,6 +1706,70 @@ <h3>Validity Period</h3>
 valid from the time the <code>verifiable credential</code> was created.
         </p>
 
+        <h4>Representing Time</h4>
+
+        <p>
+When representing time values, implementers are urged to understand that times
+are not implemented in a uniform manner throughout the world. For example:
+        </p>
+
+        <ul>
+          <li>
+The observation of daylight savings time does not start or end on the same day
+and time, if at all, between various regions.
+          </li>
+          <li>
+Calendaring systems other than the Gregorian calendar are actively used by
+various cultures.
+          </li>
+          <li>
+Leap seconds might not be taken into account in all software systems, especially
+for dates and times that precede the introduction of the leap second.
+          </li>
+        </ul>
+
+        <p>
+These are just a few examples that illustrate that whether or not a time is seen
+as valid can be based on the region in which one resides. In other words, some
+dates and times exist in some regions, while they do not exist for other
+regions.
```

Comment:
This is misleading. Some **_wall time_** values do not exist in certain regions, but the timeline is continuous. Further, for `dateTimeStamp` values, it is always possible to normalize any valid value to an incremental (think `Date`) value, even if the digits in the time stamp never show on local clocks (generally because of DST/ST rules).

For example, in `America/Los_Angeles`, DST started at `2023-03-12T02:00:00-08:00` (the next second was `2023-03-12T03:00:01-07:00` in local time). However, it is perfectly possible to interpret `2023-03-12T02:00:01-08:00` to be the same moment in time because an offset is not a time zone.

Reply:
Fixed in 00ee176d6df204e879e02e36c360b817d0a4d1ee.